setup.py

```diff
@@ -42,37 +44,43 @@
     'pytest-cov',
 ]
 
-data_spec = [
+data_files_spec = [
     # Lab extension installed by default:
     ('share/jupyter/lab/extensions',
      'lab-dist',
-     'multicontentsmanager-*.tgz'),
+     'jupyter-fs-*.tgz'),
     # Config to enable server extension by default:
     ('etc/jupyter',
      'jupyter-config',
      '**/*.json'),
 ]
 
+def runPackLabextension():
+    if Path('package.json').is_file():
+        try:
+            run(['jlpm', 'build:all'])
+        except CalledProcessError:
+            pass
+pack_labext = command_for_func(runPackLabextension)
 
-cmdclass = create_cmdclass('js', data_files_spec=data_spec)
-cmdclass['js'] = combine_commands(
-    install_npm(here, build_cmd='build:all'),
+cmdclass = create_cmdclass('pack_labext', data_files_spec=data_files_spec)
+cmdclass['pack_labext'] = combine_commands(
+    command_for_func(runPackLabextension),
     ensure_targets([
-        pjoin(here, 'lib', 'index.js'),
-        pjoin(here, 'style', 'index.css')
+        path.join(here, 'lib', 'index.js'),
+        path.join(here, 'style', 'index.css')
     ]),
 )
-
+cmdclass.pop('develop')
 
 setup(
     name=name,
     version=version,
     description='A Filesystem-like mult-contents manager backend for Jupyter',
     long_description=long_description,
     long_description_content_type='text/markdown',
-    url='https://github.com/timkpaine/jupyter-fs',
-    author='Tim Paine',
-    author_email='t.paine154@gmail.com',
+    url='https://github.com/jpmorganchase/jupyter-fs',
+    author='Tim Paine<t.paine154@gmail.com>, Max Klein <telamonian@hotmail.com>',
```

Comment:
change to "jupyter-fs authors" to match the copyright notice (`# Copyright (c) 2019, the jupyter-fs authors.`). Add your email to AUTHORS in the top level.

Reply:
sure

Reply:
done